components/automate-ui/src/app/pages/team-management/team-management.component.html

```diff
@@ -3,35 +3,46 @@
 
 <div class="container">
   <main>
+    <chef-page-header>
+      <chef-heading>Teams</chef-heading>
+      <chef-subheading>Chef Automate only displays local teams.</chef-subheading>
+    </chef-page-header>
+
+    <app-create-object-modal
+      [visible]="createModalVisible"
+      [creating]="creatingTeam"
+      [conflictErrorEvent]="conflictErrorEvent"
+      objectNoun="team"
+      [createForm]="createTeamForm"
+      (close)="closeCreateModal()"
+      (createClicked)="createTeam()">
+    </app-create-object-modal>
+
     <app-delete-object-modal
-      [visible]="modalVisible"
+      [visible]="deleteModalVisible"
       objectNoun="team"
       [objectName]="(iamMajorVersion$ | async) === 'v1' ? teamToDelete?.id : teamToDelete?.name"
```

Comment:
Hmm it's used so often, we could introduce helper functions, I guess: `isV1()`, `isV2()`.